enatega-multivendor-app/src/screens/Menu/Menu.js

```
@@ -545,71 +440,54 @@ function Menu({ route, props }) {
+  })
+
   const applyFilters = () => {
-    let filteredData =
-      queryType === 'orderAgain'
-        ? [...data?.recentOrderRestaurantsPreview]
-        : queryType === 'topPicks'
-          ? [...data?.mostOrderedRestaurantsPreview]
-          : queryType === 'topBrands'
-            ? [...data?.topRatedVendorsPreview]
-            : [...data?.nearByRestaurantsPreview?.restaurants]
+    let filteredData = queryType === 'orderAgain' ? [...data?.recentOrderRestaurantsPreview] : queryType === 'topPicks' ? [...data?.mostOrderedRestaurantsPreview] : queryType === 'topBrands' ? [...data?.topRatedVendorsPreview] : [...data?.nearByRestaurantsPreview?.restaurants]
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)